Composer/packages/lib/code-editor/demo/src/index.tsx

```diff
@@ -1,6 +1,14 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
 import React from 'react';
 import ReactDOM from 'react-dom';
 
-import App from './App';
+import App from './lgEditor';
+// import App from './jsonEditor';
```

Comment:
what's going on here?

Reply:
Here I want add more test scenario, `lgEditor` `luEditor` `inlineEditor`. Maybe a index page would be better

Reply:
If the code is commented and is not used, we should remove them.